src/code/z_play.c

```diff
@@ -1157,11 +1157,11 @@ void Play_Draw(PlayState* this) {
         gSPSegment(POLY_OPA_DISP++, 0x01, this->billboardMtx);
 
         if (!OOT_DEBUG || (R_HREG_MODE != HREG_MODE_PLAY) || R_PLAY_DRAW_COVER_ELEMENTS) {
-            Gfx* gfxP;
-            Gfx* sp1CC = POLY_OPA_DISP;
+            Gfx* tempGfxDisp;
+            Gfx* lockedGfxDisp;
 
-            gfxP = Gfx_Open(sp1CC);
-            gSPDisplayList(OVERLAY_DISP++, gfxP);
+            tempGfxDisp = Gfx_Open(lockedGfxDisp = POLY_OPA_DISP);
```

Comment:
I'm not convinced by the temp/locked naming scheme (nor "disp")

Reply:
The `Gfx_Open(lockedGfxDisp = POLY_OPA_DISP);` is awful too

Reply:
`Gfx_Open(lockedGfxDisp = POLY_OPA_DISP);` is unusual yes, but I'd argue that it makes sense to write unusual code to force POLY_OPA_DISP to appear inside the call to Gfx_Open in this case. If you think of the high level picture, that call to Gfx_Open is creating a side effect on POLY_OPA_DISP, it's taking all of the memory it has and giving it to something else. 
When you have the assignment above Gfx_Open I feel that you lose a little bit of that connection.